app/src/renderer/app.css

```
@@ -0,0 +1,38 @@
+  --surface-panel: #0e131b;
+  --surface-elevated: #131b27;
+  --line-soft: rgba(171, 188, 210, 0.12);
+  --line: rgba(171, 188, 210, 0.2);
+  --line-strong: rgba(214, 252, 194, 0.32);
+  --text-primary: #eef5ff;
+  --text-secondary: #b6c1cf;
+  --text-muted: #7f8b99;
+}
+
+@theme {
+  --font-display: "Azeret Mono", "SF Mono", Menlo, monospace;
```

Stylelint (17.3.0)

[error] Expected "Menlo" to be "menlo" (value-keyword-case)

(value-keyword-case)